content/en/user-guide/lambda-tools/lambda-vscode-extension/index.md

```diff
@@ -1,8 +1,10 @@
 ---
 title: "Lambda VSCode Extension"
```

Comment:
IMHO, the "LocalStack VSCode Extension" would fit better under "LocalStack Tools". In its current state, it is indeed very limited and only relevant for Lambda, I guess that's why it ended up under the "Lambda Tools":

<img width="292" alt="Screenshot 2024-01-02 at 19 14 06" src="https://github.com/localstack/docs/assets/8161292/474b9e29-fd89-4e71-b1d0-5ed73b4e9079">

Reply:
I believe we can still keep it around in `Lambda Tools` given that it only focusses on improving the Lambda Development experience right now.